### DISCUSSION TITLE
Ord.exe not opening

### DISCUSSION THREAD

**Discussion by FlashLoanArbs** [ordinals/ord#1634] [Q&A] [answered]
Bitcoin node is synced, ord release for Windows is downloaded on Windows 10 and extracted, yet ord doesn't open when I try. 
Anybody have any ideas on what I'm doing wrong? 
At first I got the "vcruntime140.dll not found" error, but i downloaded a C++ package and no longer got that error.
But now nothing opens at all.
Is there command line functionality for windows? 'bitcoind -version' is not recognized by windows. 
Sorry for the noob questions, just trying to figure all this out and there isn't much documentation at all out there.
I appreciate any help that you can offer.
Mike

**Comment by goddesslens** [comment, marked as answer]
Open cmd,
Navigate to bitcoind folder. Run bitcoind using bitcoind -txindex
Keep it open. Do not close bitcoind; Now you can run ord wallet by opening another cmd tab(Navigate to ord wallet on cmd)

**Reply by FlashLoanArbs** [reply]
You rock dude! Thank you.
